csfieldguide/chapters/models.py

```diff
@@ -71,6 +73,24 @@ def __str__(self):
         """
         return self.heading
 
+    def clean(self):
```

Comment:
Awesome to see this, especially as it leverages Django (I learnt new stuff!)

Reply:
there is actually an even more awesome way to do it, using validators, but ran into quite a bit of trouble trying to implement it so this was a good plan B, just not quite as cool. (both are as good as each other to use/I don't think there is a good practice bias, I did a bit of reading).